Source/DafnyLanguageServer.Test/Various/CounterExampleTest.cs

```diff
@@ -948,10 +948,10 @@ method T_map0(m:map<int,int>, key:int, val:int)
     [TestMethod]
     public async Task MapsValues() {
       var source = @"
-      method test(m:map<int,char>) {
-       var values := m.Values;
-       assert ('c' !in values);
-      }".TrimStart();
+method test(m:map<int,char>) {
```

Comment:
The indentation looks weird here. Can you please double-check?

Reply:
Christoph asked me to keep the indentation this way for continuity reasons (here is a [link to a comment about it](https://github.com/dafny-lang/dafny/pull/1336#discussion_r684984964)) earlier. That said, the majority of the tests in this file are now written for the new versions of the tool, so perhaps it makes sense to fix the indentation. Let me know and I will make the fix!